network/network.go

```diff
@@ -0,0 +1,68 @@
+package network
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/dedis/cothority/monitor"
+)
+
+// Conn is the basic interface to represent any communication mean
+// between two host.
+type Conn interface {
+	// Send a message through the connection.
+	// obj should be a POINTER to the actual struct to send, or an interface.
+	// It should not be a Golang type.
+	Send(ctx context.Context, obj Body) error
```

Comment:
Ok... I'm still hesitating about that because I'm afraid we **should** use this `Context`, rather than say "if we don't use it, then lets drop it"...
but since you're the majority, I abide :)